### PR TITLE
Add Azure CLI command to Airlock Files Section

### LINKED ISSUE #2916: Add storage explorer link and az cli command to Airlock UI request storage link section
**Is your feature request related to a problem? Please describe.**
A couple of users (i.e #2897) have tried to use the SAS URL directly in a web browser as there isn't much guidance in the UI on what a SAS URL is and how to use it (i.e. pasting it into Azure Storage explorer or using the az cli).

**Describe the solution you'd like**
Instead of expecting all users to have reviewed the documentation, we should make it clearer in the UI by offering drop-down options for not just the raw SAS URL, but a storage Explorer web link and an az cli command - giving users options out of the box on how to consume the SAS URL.

### PR DESCRIPTION
**Pull request by yuvalyaron**
# Resolves #2916 

## What is being addressed

Added the option to upload/download files to airlock requests using an Azure CLI command.

## How is this addressed

- Added tabs for selecting the option to access the blob storage, moved the existing logic (SAS URL generation), along with the new code to a new component - `AirlockRequestFilesSection`
- Added a new component for displaying and copying Azure CLI commands - `CliCommand`
- included dependabot's PR #3192 

image

### CODE REVIEW

File: ui/app/src/components/shared/airlock/AirlockViewRequest.tsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
-import { DefaultButton, Dialog, DialogFooter, DocumentCard, DocumentCardActivity, DocumentCardDetails, DocumentCardTitle, DocumentCardType, FontIcon, getTheme, IStackItemStyles, IStackStyles, IStackTokens, mergeStyles, MessageBar, MessageBarType, Modal, Panel, PanelType, Persona, PersonaSize, PrimaryButton, Spinner, SpinnerSize, Stack, TextField } from "@fluentui/react";
```
Comment (yuvalyaron) [inline ui/app/src/components/shared/airlock/AirlockViewRequest.tsx:1]: the deleted code here was moved without modifications to AirlockRequestFilesSection

File: ui/app/yarn.lock [hunk 2]
```diff
@@ -5434,9 +5434,9 @@ htmlparser2@^6.1.0:
     entities "^2.0.0"
 
 http-cache-semantics@^4.1.0:
-  version "4.1.0"
-  resolved "https://registry.yarnpkg.com/http-cache-semantics/-/http-cache-semantics-4.1.0.tgz#49e91c5cbf36c9b94bcfcd71c23d5249ec74e390"
-  integrity sha512-carPklcUh7ROWRK7Cv27RPtdhYhUsela/ue5/jKzjegVvXDqM2ILE9Q2BGn9JZJh1g87cp56su/FgQSzcWS8cQ==
+  version "4.1.1"
+  resolved "https://registry.yarnpkg.com/http-cache-semantics/-/http-cache-semantics-4.1.1.tgz#abe02fcb2985460bf0323be664436ec3476a6d5a"
+  integrity sha512-er295DKPVsV82j5kw1Gjt+ADA/XYHsajl82cGNQG2eyoPkvgUhX+nDIyelzhIWbbsXP39EHcI6l5tYs2FYqYXQ==
```
Comment (yuvalyaron) [inline ui/app/yarn.lock:5440]: https://github.com/microsoft/AzureTRE/pull/3192

**Review by jjgriff93 — APPROVED** [review]
Brilliant work - love the tabs, nice way to keep it neat and the copy functionality is really nice. Couple of thoughts:

- Since switching to the CLI tab calls the generate link endpoint, might it be best to merge the `generateFilesLink` and `generateAzureCliCommand` and once either the generate button or switching the tab has generated it, it isn't called again? For example, clicking the CLI tab, then switching back to the SAS URI, I think we should already have the link there instead of the user having to click generate
- Your copy button is really nice, with the "Copied" pop-up after clicking - would be great to add that to the copy button in the SAS URI section if it's not too much trouble?

**Comment by yuvalyaron** [comment]
> Brilliant work - love the tabs, nice way to keep it neat and the copy functionality is really nice. Couple of thoughts:
> 
> * Since switching to the CLI tab calls the generate link endpoint, might it be best to merge the `generateFilesLink` and `generateAzureCliCommand` and once either the generate button or switching the tab has generated it, it isn't called again? For example, clicking the CLI tab, then switching back to the SAS URI, I think we should already have the link there instead of the user having to click generate
> * Your copy button is really nice, with the "Copied" pop-up after clicking - would be great to add that to the copy button in the SAS URI section if it's not too much trouble?

Fixed both :)

**Pull request merged**